tools/cleanlab/cleanlab_issue_handler.xml

```diff
@@ -0,0 +1,199 @@
+<tool id="cleanlab_issue_handler" name="Cleanlab Issue Handler" version="1.1.1" profile="23.0">
```

Comment:
should the version be 2.7.1? If so please use a macro and reuse this version in the two places.

Reply:
See example here: https://github.com/bgruening/galaxytools/blob/master/tools/tabpfn/tabpfn.xml#L1C73-L1C85

`@TOOL_VERSION@+galaxy@VERSION_SUFFIX@`

@TOOL_VERSION can take the version of the Cleanlab and @VERSION_SUFFIX can take tool's version

for example: @TOOL_VERSION == 2.7.1 and @VERSION_SUFFIX is 1.0

then, the combined tool's version becomes `2.7.1+galaxy1.0`